src/platform/packages/shared/kbn-esql-ast/src/parser/walkers.ts

```diff
@@ -222,7 +222,7 @@ function visitLogicalAndsOrs(ctx: LogicalBinaryContext) {
 }
 
 function visitLogicalIns(ctx: LogicalInContext) {
-  const fn = createFunction(ctx.NOT() ? 'not_in' : 'in', ctx, undefined, 'binary-expression');
+  const fn = createFunction(ctx.NOT() ? 'not in' : 'in', ctx, undefined, 'binary-expression');
```

Comment:
I changed the operators to not use underscore following `is null` and `is not null` ast. It also simplifies some things